website/blog/2022-05-03-automatic-mode/index.md

```diff
@@ -0,0 +1,28 @@
+---
+slug: automatic-mode
+title: Announcing Automatic Mode
+authors: [nvh95, thanhsonng]
```

Comment:
Thank you for including my name even though you rebased my commits

Reply:
I will find justice for you one day. I promise...